lib/Dialect/QUIR/Transforms/AngleConversion.cpp

```diff
@@ -101,6 +101,9 @@ void QUIRAngleConversionPass::runOnOperation() {
   mlir::GreedyRewriteConfig config;
   // Disable to improve performance
   config.enableRegionSimplification = false;
+  config.strictMode = mlir::GreedyRewriteStrictness::ExistingOps;
+  // Each operation can only be modified once so limit
+  config.maxIterations = 1;
```

Comment:
Of these two changes, which provides more benefit? I'm surprised that the first changes things at all since the rewrite pattern isn't adding any new operations.

Reply:
I think its maxIterations